lib/ansible/modules/crypto/openssh_cert.py

```diff
@@ -31,11 +31,13 @@
         choices: [ 'present', 'absent' ]
         description:
             - Whether the host or user certificate should exist or not, taking action if the state is different from what is stated.
+        type: str
     type:
         required: true
         choices: ['host', 'user']
         description:
             - Whether the module should generate a host or a user certificate.
+        type: str
```

Comment:
We tend to sort these keys as well. It helps to scan parameters if the same order is used.
The order is: `description`, `type`, `required`, `choices`, `default`, `added_in_version`